src/auth/forward.h

```diff
@@ -25,6 +25,8 @@ typedef std::vector<Auth::SchemeConfig *> ConfigVector;
 
 class UserRequest;
 
+using Ttl = int32_t;
```

Comment:
I am worried that the suggested name -- "Auth::Ttl" -- is too general because authentication code may need several different types for TTL. For example, `authenticate_ttl` uses a different type today. However, I assume you consider such worries unfounded and am not requesting any related changes.